INSTALL.md

```diff
@@ -0,0 +1,67 @@
+# How to install MacGitver
+
+In present state, you need to compile MacGitver from source. We provide a compilable version from our default branch `development`. If you like to help with creating binary packages, please create a [pull request](https://github.com/macgitver/MacGitver/pulls) or contact me via [mail](mailto:nils@macgitver.org).
+
+## Prerequisites
```

Comment:
It's a recent Lubuntu 15.04. The only "-dev" package I see is `libqt5opengl5-dev`. See for yourself:

```bash
nils@nils-lubuntu-mac:~/Projects/libGitWrap$ dpkg --list | grep libqt5
rc  libqt53d5:amd64                             5.0~git20130731-0ubuntu10                      amd64        Qt 3D module
rc  libqt5clucene5:amd64                        5.2.1-8build1                                  amd64        Qt 5 CLucene module
ii  libqt5concurrent5:amd64                     5.4.1+dfsg-2ubuntu4                            amd64        Qt 5 concurrent module
ii  libqt5core5a:amd64                          5.4.1+dfsg-2ubuntu4                            amd64        Qt 5 core module
ii  libqt5dbus5:amd64                           5.4.1+dfsg-2ubuntu4                            amd64        Qt 5 D-Bus module
rc  libqt5declarative5:amd64                    5.2.1-1ubuntu1                                 amd64        Qt Quick 1 module for Qt 5
rc  libqt5designer5:amd64                       5.2.1-8build1                                  amd64        Qt 5 designer module
rc  libqt5designercomponents5:amd64             5.2.1-8build1                                  amd64        Qt 5 Designer components module
ii  libqt5gui5:amd64                            5.4.1+dfsg-2ubuntu4                            amd64        Qt 5 GUI module
rc  libqt5help5:amd64                           5.2.1-8build1                                  amd64        Qt 5 help module
ii  libqt5location5:amd64                       5.4.1-1ubuntu1                                 amd64        Qt Location module
rc  libqt5multimedia5:amd64                     5.3.0-1ubuntu2                                 amd64        Qt 5 Multimedia module
rc  libqt5multimediaquick-p5:amd64              5.3.0-1ubuntu2                                 amd64        Qt 5 Multimedia Quick module
rc  libqt5multimediawidgets5:amd64              5.3.0-1ubuntu2                                 amd64        Qt 5 Multimedia Widgets module
ii  libqt5network5:amd64                        5.4.1+dfsg-2ubuntu4                            amd64        Qt 5 network module
ii  libqt5opengl5:amd64                         5.4.1+dfsg-2ubuntu4                            amd64        Qt 5 OpenGL module
ii  libqt5opengl5-dev:amd64                     5.4.1+dfsg-2ubuntu4                            amd64        Qt 5 OpenGL library development files
ii  libqt5positioning5:amd64                    5.4.1-1ubuntu1                                 amd64        Qt Positioning module
ii  libqt5printsupport5:amd64                   5.4.1+dfsg-2ubuntu4                            amd64        Qt 5 print support module
ii  libqt5qml5:amd64                            5.4.1-1ubuntu5                                 amd64        Qt 5 QML module
ii  libqt5quick5:amd64                          5.4.1-1ubuntu5                                 amd64        Qt 5 Quick library
rc  libqt5quickparticles5:amd64                 5.2.1-3ubuntu15.1                              amd64        Qt 5 Quick particules module
rc  libqt5quicktest5:amd64                      5.2.1-3ubuntu15.1                              amd64        Qt 5 Quick Test library
ii  libqt5scintilla2-11                         2.8.4+dfsg-1build0.1                           amd64        Qt5 port of the Scintilla source code editing widget
ii  libqt5scintilla2-l10n                       2.8.4+dfsg-1build0.1                           all          Scintilla source code editing widget for Qt5, translation files
ii  libqt5script5:amd64                         5.4.1+dfsg-3                                   amd64        Qt 5 script module
rc  libqt5sensors5:amd64                        5.3.0+dfsg-1ubuntu2                            amd64        Qt Sensors module
ii  libqt5sql5:amd64                            5.4.1+dfsg-2ubuntu4                            amd64        Qt 5 SQL module
ii  libqt5sql5-sqlite:amd64                     5.4.1+dfsg-2ubuntu4                            amd64        Qt 5 SQLite 3 database driver
ii  libqt5svg5:amd64                            5.4.1-1                                        amd64        Qt 5 SVG module
ii  libqt5test5:amd64                           5.4.1+dfsg-2ubuntu4                            amd64        Qt 5 test module
ii  libqt5webkit5:amd64                         5.4.1+dfsg-2ubuntu1                            amd64        Web content engine library for Qt
ii  libqt5widgets5:amd64                        5.4.1+dfsg-2ubuntu4                            amd64        Qt 5 widgets module
ii  libqt5x11extras5:amd64                      5.4.1-1                                        amd64        Qt 5 X11 extras
ii  libqt5xdg-data                              1.2.0+bzr181+201505042156~ubuntu15.04.1        all          libqt5xdg-data
rc  libqt5xdg0                                  0.5.3+bzr128+201408282130~ubuntu14.04.1        amd64        libqt5xdg0
ii  libqt5xdg1                                  1.2.0+bzr181+201505042156~ubuntu15.04.1        amd64        libqt5xdg1
ii  libqt5xml5:amd64                            5.4.1+dfsg-2ubuntu4                            amd64        Qt 5 XML module
```